package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "express-generator",
   "description": "Express' application generator",
-  "version": "4.13.1",
+  "version": "4.14.0",
```

Comment:
There is not even a 4.14 line of Express yet, so bumping wouldn't even make sense (and version bumps are done in their own commit, when a version is cut @tunniclm already wrote about this in the pending release documentation).